doc/source/development/pandas_development_faq.rst

```diff
@@ -0,0 +1,217 @@
+.. _pandas_development_faq:
+
+{{ header }}
+
+======================
+Pandas Development FAQ
+======================
+
+.. contents:: Table of contents:
+   :local:
+
+Purpose
+=======
+
+Based on https://github.com/pydata/pandas/pull/4404#issuecomment-22864665 this
```

Comment:
This link is outdated, we're in the pandas-dev organization now. Also, you can make it a link with rst syntax.

Reply:
I fix it.